BraintreeCore/src/main/java/com/braintreepayments/api/Configuration.kt

```diff
@@ -117,37 +85,197 @@ open class Configuration internal constructor(configurationString: String?) {
     // endregion
 
     // region Internal Properties
+
+    /**
+     * @return [String] url of the Braintree analytics service.
+     * @suppress
+     */
```

Comment:
Made all restricted property annotations inline to allow them to be hidden in the docs.